ai_diffusion/workflow.py

```diff
@@ -11,6 +13,14 @@
 from .util import client_logger as log
 
 
+_pattern_lora = re.compile(r"\s*<lora:([^:<>]+)(?::(-?[^:<>]*))?>\s*", re.IGNORECASE)
+
+
+class LoraException(Exception):
```

Comment:
Just use a basic `Exception` (unless you add extra fields or use it for type filtering)